### DISCUSSION TITLE
Vacuum card config help needed

### DISCUSSION THREAD

**Discussion by ntimo** [hsakoh/switchbot-mqtt#43] [Q&A]
Hello,
Do you maybe by any chance use https://github.com/denysdovhan/vacuum-card and would be able to help me to configure it for a SwitchBot vacuum, I can't seem to get it to work. 

Best wishes and thanks,
Timo

**Comment by zalmael** [comment]
Hi, this is normal, cause the devices that come from the add-on are NOT considered as a vacuum in home assistant, but a bunch of selects, buttons....
You need to create it using templates. I'm currently in the process of building it, I can share my code when done, but it will not have all the settings available in a "classic" vacuum entity. You can see how to do it on this page : https://www.home-assistant.io/integrations/vacuum.template/
Edit : Or read this discussion https://github.com/hsakoh/switchbot-mqtt/discussions/23 :)
